domain/model/modelmigration/import_test.go

```diff
@@ -216,7 +216,7 @@ func (i *importSuite) TestModelCreateRollbacksOnFailure(c *gc.C) {
 		UUID:  modelUUID,
 	}
 
-	finalised := false
+	var finalised bool
```

Comment:
Change all of this nomenclature to be "activator", "activated" etc. I've already asked Tom to do this.